x-pack/plugins/security_solution/public/assistant/use_conversation_store/index.tsx

```diff
@@ -7,18 +7,26 @@
 
 import type { Conversation } from '@kbn/elastic-assistant';
 
+import { unset } from 'lodash/fp';
+import { DATA_QUALITY_DASHBOARD_CONVERSATION_ID } from '@kbn/ecs-data-quality-dashboard/impl/data_quality/data_quality_panel/tabs/summary_tab/callout_summary/translations';
 import { useLocalStorage } from '../../common/components/local_storage';
 import { LOCAL_STORAGE_KEY } from '../helpers';
 import { BASE_SECURITY_CONVERSATIONS } from '../content/conversations';
+import { useLinkAuthorized } from '../../common/links';
+import { SecurityPageName } from '../../../common';
 
 export interface UseConversationStore {
   conversations: Record<string, Conversation>;
   setConversations: React.Dispatch<React.SetStateAction<Record<string, Conversation>>>;
 }
 
 export const useConversationStore = (): UseConversationStore => {
+  const isDataQualityDashboardPageExists = useLinkAuthorized(SecurityPageName.dataQuality);
+  const baseConversations = isDataQualityDashboardPageExists
+    ? BASE_SECURITY_CONVERSATIONS
+    : unset(DATA_QUALITY_DASHBOARD_CONVERSATION_ID, BASE_SECURITY_CONVERSATIONS);
```

Comment:
Love it! 😉 
Can we wrap `baseConversations` in a _useMemo_? `unset` returns a new object every time, we should call it only when `isDataQualityDashboardPageExists` changes.